src/main/webapp/WEB-INF/tags/instructor/results/questionPanel.tag

```diff
@@ -87,7 +87,7 @@
               </thead>
               <tbody>
                 <c:forEach items="${questionPanel.responses}" var="responseRow" varStatus="status">
-                  <results:responseRow responseRow="${responseRow}" questionIndex="${questionIndex}"
+                  <results:responseRow responseRow="${responseRow}" questionIndex="${questionPanel.question.questionNumber}"
```

Comment:
`questionIndex="${questionIndex}"` was added in entering comments from Question view PR. I think it is wrong way to get question index because panels are loaded by ajax and `${questionIndex}` was getting calculated by iterations.
If you see the file https://github.com/TEAMMATES/teammates/blob/e523242f0eeb641005943729870da0a90d30f766/src/test/resources/pages/instructorFeedbackResultsPageOpen.html?utf8=%E2%9C%93 and search `data-target="#commentModal-2-1-0"`, you will get 4 matches instead of 1. So basically comment modals were not correctly associated in Questions View.

Reply:
> So basically comment models were not correctly associated in Questions View.

Good observation!